site/src/components/SSHButton/SSHButton.tsx

```diff
@@ -83,9 +89,47 @@ export const SSHButton: React.FC<React.PropsWithChildren<SSHButtonProps>> = ({
         </Stack>
 
         <HelpTooltipLinksGroup>
-          <HelpTooltipLink href="https://coder.com/docs/coder-oss/latest/install">
-            Install Coder CLI
-          </HelpTooltipLink>
+          <Link
+            onClick={() => setDownloadCLIOpen(!downloadCLIOpen)}
+            className={styles.link}
+          >
+            {downloadCLIOpen ? (
+              <ExpandMoreIcon className={styles.linkIcon} />
+            ) : (
+              <ChevronRightIcon className={styles.linkIcon} />
+            )}
+            <Typography
+              className={styles.downloadCLIButton}
+              onClick={() => setDownloadCLIOpen(!downloadCLIOpen)}
+            >
+              Download Coder CLI
+            </Typography>
+          </Link>
+          <Maybe condition={downloadCLIOpen}>
+            <div className={styles.downloadCLIOptionsDiv}>
+              <HelpTooltipLink href="/bin/coder-windows-amd64.exe">
+                For Windows (AMD64)
+              </HelpTooltipLink>
+              <HelpTooltipLink href="/bin/coder-windows-arm64.exe">
+                For Windows (ARM)
+              </HelpTooltipLink>
+              <HelpTooltipLink href="/bin/coder-darwin-amd64">
+                For Mac OS (Intel)
+              </HelpTooltipLink>
+              <HelpTooltipLink href="/bin/coder-darwin-arm64">
+                For Mac OS (ARM)
+              </HelpTooltipLink>
+              <HelpTooltipLink href="/bin/coder-linux-amd64">
+                For Linux (AMD64)
+              </HelpTooltipLink>
+              <HelpTooltipLink href="/bin/coder-linux-arm64">
+                For Linux (ARM64)
+              </HelpTooltipLink>
+              <HelpTooltipLink href="/bin/coder-linux-arm64">
+                For Linux (ARMv7)
+              </HelpTooltipLink>
```

Comment:
This should come from some typed constant array like:
```
interface CoderSlimBin {
  os: "windows" | "linux" | "darwin"
  arch: "amd64" | "arm64" | "armv7"
}

const SlimBins: CoderSlimBin[] = [
  ...
]
```

Then use a function to generate the filename and the text.